helm-charts/Makefile

```diff
@@ -0,0 +1,52 @@
+PROJECT_NAME=helm-charts
+
+RELEASE_VERSION ?= latest
+CHART_SEMANTIC_RELEASE_VERSION ?= $(shell cat ../release.version | sed 's/\([0-9.]*\).*/\1/')
+CHART_NAME=strimzi-kafka-operator
+CHART_PATH=./$(CHART_NAME)/
+CHART_RENDERED_TEMPLATES_TMP=../target/charts
+CHART_RENDERED_TEMPLATES_EXAMPLES=../examples/install/cluster-operator/
+
+helm_clean:
+	rm -rf $(CHART_RENDERED_TEMPLATES_TMP)
+	rm -rf strimzi-$(RELEASE_VERSION)/charts/
+	#rm -rf $(CHART_RENDERED_TEMPLATES_EXAMPLES)
+	rm -f $(CHART_NAME)-*.tgz
+
+helm_lint:
+	# Linting Helm Chart
+	helm lint --debug $(CHART_PATH)
+
+helm_template:
+	# Generating rendered template files to: $(CHART_RENDERED_TEMPLATES_TMP)
+	mkdir -p $(CHART_RENDERED_TEMPLATES_TMP)
+	helm template --namespace myproject --output-dir $(CHART_RENDERED_TEMPLATES_TMP) --set imageTagOverride=$(RELEASE_VERSION) $(CHART_PATH)
```

Comment:
Do developers need to change the namespace when it's different from "myproject" (i.e. "default" in K8S) ? But because Helm is about K8S why we have "myproject" here?

Reply:
We're using Helm template here to generate the resources in `examples`. Thus either "myproject" or "default" would be as wrong it always has been in the examples for whichever is the one whose default we don't chose. In other words: This is no different than it's always been.